Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Node.run(Node.java:141)
	at Node.main(Node.java:248)
